### PR TITLE
Style blog based on Yunus's design

### PR DESCRIPTION
**Pull request by JolandaNava**
Still missing: 
+ styling of code blocks for blogposts
+ illustrations from Yunus (already sent him the project descriptions)

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,41 +1,6 @@
...
-          }
-        } else if (currentTop > this.previousTop) {
-          //if scrolling down...
-          $('#mainNav').removeClass('is-visible');
-          if (currentTop > headerHeight && !$('#mainNav').hasClass('is-fixed')) $('#mainNav').addClass('is-fixed');
-        }
-        this.previousTop = currentTop;
-      });
-  }
+  // add code here
```
Comment (chshersh) [inline js/main.js]: Is this file needed now if it doesn't contain anything?

File: templates/post.html [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
-    <div class="row tag-block text-center">
-        <div class="col-md-12 col-xs-12 tag-block">
-            $for(tagsList)$
-            <span> <a class="btn btn-outline-dark" href="../tags/$tag$">$tag$</a></span>
-            $endfor$
+    <div class="row">
+        <div class="col-lg-8 col-md-10 mx-auto">
+            <h2 class="title">$title$</h2>
+            <div class="post-context">
+              <span class="author">Author Here | </span>
```
Comment (chshersh) [inline templates/post.html]: If I remember, we decided to not mention the author name, so this part can be removed

**Review by chshersh — COMMENTED** [review]
Looks really good!

Regarding code highlight: we don't have any code right now, so it's okay to not have code highlight integration right now. Can be done in a separate PR when some blog post with code appears.

**Review by vrom911 — APPROVED** [review]
Looks great! ❤️ 
I just wonder if it's intentional that the font for the blog post text and font for blockquotes in there are different?

**Review by chshersh — APPROVED** [review]
